estate/models/res_users.py

```diff
@@ -0,0 +1,6 @@
+from odoo import fields,models 
+
+class ResUsers(models.Model):
+    _inherit='res.users'
+    
+    property_ids = fields.One2many('estate.property',inverse_name='seller_id',domain="([ ('status', 'in', ('new','offer_received'))])")
```

Comment:
```suggestion
    property_ids = fields.One2many('estate.property', inverse_name='seller_id', domain="([('status', 'in', ('new','offer_received'))])")
```
why you have applied domain here?

Reply:
user only have to only manage the properties which is new or offer received